cmake/std/atdm/load-env.sh

```diff
@@ -5,9 +5,8 @@
 ################################################################################
 
 # Assert this script is sourced, not run!
-called=$_
-if [ "$called" == "$0" ] ; then
-  echo "This script '$0' is being called.  Instead, it must be sourced!"
+if [ "${BASH_SOURCE[0]}" == "${0}" ] ; then
+  echo "This script '${0}' is being called.  Instead, it must be sourced!"
```

Comment:
In Scot’s patch, the curly braces weren’t there around the 0.  Not sure if that changes things.

Reply:
Scot's patch was:

```
-if [ "$called" == "$0" ] ; then
+if [ "${BASH_SOURCE[0]}" == "${0}" ] ; then
   echo "This script '$0' is being called.  Instead, it must be sourced!"
   exit 1
 fi
```

I just added the `{}` around `0` in the `echo` statement to be consistent.

Reply:
@jmgate, can you approve?